src/wazuh_db/wdb_integrity.c

```diff
@@ -469,3 +470,41 @@ int wdbi_query_clear(wdb_t * wdb, wdb_component_t component, const char * payloa
 
     return ret_val;
  }
+
+int wdbi_check_sync_status (wdb_t *wdb, wdb_component_t component) {
```

Comment:
Remove the space after the function name.

Reply:
Fixed!